### ISSUE TITLE
Add the ability to record sessions and send them by email via private SMTP servers

### ISSUE THREAD

**Issue by emaalouf** [cracker0dks/Accelerator#36]
I think that recording sessions is of top priority as it is something essential for users to rewatch the session, let's also work on that what do you say?

**Comment by cracker0dks** [comment]
Yeah I thought about that as well some time ago. The best way I see is to start a screenshare with audio enabled and send this as stream (or data only) to the server to save.
If we only record the video/audio from people we miss the presentation and stuff.
The problem is, that the recording relies on the person starting the recording.

**Comment by dsingh1212** [comment]
We can start with recording to start after initiated by the host/presenter. This way we will have at least a way to record, save and share the session.
